infra/src/main/resources/db/migration/repeatable/R__10.04_layout_graph_view.sql

```diff
@@ -1,33 +1,79 @@
 drop view if exists layout.location_track_version_switch_view;
 create view layout.location_track_version_switch_view as
-select distinct
-  ltve.location_track_id,
-  ltve.location_track_layout_context_id,
-  ltve.location_track_version,
-  np.switch_id,
-  np.switch_joint_number,
-  np.switch_joint_role,
-  -- Generate a sortable ordering for the links, based on how they connect to the edge
-  -- Note: the following edge will likely have share a node (and hence 2 links) so the distinct above is required
-  -- However, it does not matter which edge's version of the ordering number gets picked, it's still in the same place
+select
```

Comment:
Hakua puhtaasti optimoitu. Tämä "distinct" taisi olla jotain vanhempaa perua, koska tässähän haettiin jo valmiiksi ennen erikseen ensimmäisen kaaren alku ja kaikkien kaarten loput, eli tästä hausta ei muutenkaan tullut samoja rivejä usempaa kertaa; "distinct"in poistamalla tämän saa kerrottua postgressillekin.